src/UI/Seller/src/app/promotions/components/promotion-edit/promotion-edit.component.html

```diff
@@ -627,6 +687,30 @@ <h6 class="mb-0" translate>ADMIN.PROMOTIONS.USAGE_LIMITS</h6>
           </div>
         </div>
       </form>
+      <div class="justify-content-end mt-3 d-none d-md-flex">
+        <delete-confirm-modal-component
+          *ngIf="!isCreatingNew"
+          buttonText="ADMIN.NAV.PROMOTION"
+          (deleteConfirmed)="handleDelete($event)"
+        >
+        </delete-confirm-modal-component>
+        <button
+          class="btn btn-primary"
+          type="submit"
+          [disabled]="isBtnDisabled()"
+          (click)="handleSave()"
+        >
+          {{ getSaveBtnText() }}
+        </button>
+        <button
+          class="btn btn-link ml-3"
+          (click)="handleDiscardChanges()"
+          *ngIf="areChanges && !isCreatingNew"
```

Comment:
Does this mean that no 'discard changes' button will be present when you are editing a resource (not creating new)?  I think that the discard button should be present for both create and edit.

Reply:
Yeah that logic didn't change, I'm just duplicating it to appear under the summary card but I like the idea of showing it for "Edit".

I would say it isn't necessary (and not sure it's even set up to function properly) on "Create". Nothing is saved so a user can navigate away if they decide to abandon their promotion.